examples/plot_results.py

```diff
@@ -18,11 +18,11 @@
 
 acc = np.asarray(acc)
 x = [i*50000/1e6 for i in range(1, len(acc))]
-y = [np.amax(acc[:i])/100.0 for i in range(1, len(acc))]
+y = [np.amax(acc[:i]) for i in range(1, len(acc) + 1)]
```

Comment:
+1 crashes for me because x and y are not the same length.

Reply:
Will fix to ensure consistency of both.